berkeley-mobile/Calendar/EventDataSource/EventCalendarEntry.swift

```diff
@@ -9,16 +9,19 @@
 import Foundation
 import UIKit
 
-/// An event on the Academic Calendar (e.g. 'First Day of Instruction').
-class AcademicCalendarEntry: CalendarEvent {
+class EventCalendarEntry: CalendarEvent, HasImage {
 
     // MARK: CalendarEvent Fields
 
+    var category: String
     var name: String
     var date: Date
     var end: Date?
     var description: String?
     var location: String?
+    var link: URL?
+    var imageURL: URL?
+    var sourceLink: URL?
```

Comment:
Small nit, but only some of these fields are from the `CalendarEvent` protocol. Also, could you clarify the docstring for `type` (lower down in this file)? It's a bit confusing now that there's also a `category` field. Maybe you could use the word "subcategory" to describe `type`.